gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -763,9 +763,15 @@ def test_{{ method_name }}_pager(transport_name: str = "grpc"):
             )),
         )
         {% endif %}
+        {% if service.version %}
+        if HAS_GOOGLE_API_CORE_VERSION_HEADER:
+            expected_metadata = tuple(expected_metadata) + (
```

Comment:
ditto comments from the ads templates on the apparent brittleness of testing with just one long tuple

Reply:
Captured in https://github.com/googleapis/gapic-generator-python/issues/2024